x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/detections_engineer.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Role } from '@kbn/security-plugin/common';
+
+export const detectionsEngineer: Omit<Role, 'name'> = {
```

Comment:
Could you make this a function that return the object instead of having it as a mutable object? (same comment for the other roles)

Reply:
https://github.com/elastic/kibana/pull/144710/commits/d901c1b84bbddd37489e405a4214e6667619e399